addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1397,14 +1397,20 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
             let elOffset = animateOnScroll ? 0 : Math.max((elHeight * this.offsetRatio), this.offsetMin);
             const state = $el.css("animation-play-state");
 
+            // We don't want to prevent animations if a cookie bar is displayed.
+            const cookieBarClasses = ["o_cookies_classic", "o_cookies_discrete", "o_cookies_popup"];
+            const isCookieBarDisplayed = $(
+                document.body.querySelector(cookieBarClasses.map(cls => `.${cls}`).join(", "))
+            ).is(":visible");
```

Comment:
Maybe I could base my selector on the id `website_cookies_bar` but it seems a bit more risky to me

Reply:
My "fear" is that a user has custom code with a custom cookie bar to which he would have given this ID (which by the way is not prefixed by `o_` :grimacing: ) and that my code changes something for him. In my opinion I'm going too far in my questioning of what could break in stable. I gonna use the ID :+1: